realm/realm-annotations-processor/src/test/resources/io/realm/AllTypesRealmProxy.java

```diff
@@ -112,7 +113,6 @@ public final AllTypesColumnInfo clone() {
         proxyState.setConstructionFinished();
     }
 
-    @Override
```

Comment:
I think we might want to consider before we constrain the value of the backlinked field.  I can't think of any problem with restricting it, as you suggest, but I just haven't had a chance to think about it carefully.  I agree that `private final RealmResults<Something> backlinks = realm.findAll()` would be pretty odd...